pom.xml

```diff
@@ -41,18 +41,18 @@
         <!-- Versions below are for several dependencies we're using in the data & ingest modules -->
         <!-- Compile dependencies -->
         <slf4j.version>1.8.0-beta4</slf4j.version>
```

Comment:
Can you add a comment that we like to align versions with some databricks runtime dependency list 
it actually looks the same as https://learn.microsoft.com/en-us/azure/databricks/release-notes/runtime/13.0
:)